app/graphql/types/icon_id_to_name_type.rb

```diff
@@ -0,0 +1,20 @@
+# Reference: https://openweathermap.org/weather-conditions#Weather-Condition-Codes-2
```

Comment:
This Commit is rather large, is there a way we could break it up?

Reply:
Sadly, there's no clear lines to split this commit down and have the program still functional, as all the changes are to transition an API that is depended on for it.